test/helpers/story/generator/story-generator.ts

```diff
@@ -1,58 +1,121 @@
-  let manualScenarios: TestScenario[] = [
-    {
-      domain: 'LOAN',
-      actions: [
-        {
-          actionType: STORY_ACTIONS.LOAN.TAKE_OUT,
-          suiteName: '',
-          args: { pass: true, parent: null },
-        },
-        {
-          actionType: STORY_ACTIONS.LOAN.LIQUIDATE,
-          suiteName: '',
-          args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT },
-        },
-        {
-          actionType: STORY_ACTIONS.LOAN.REPAY,
-          suiteName: '',
-          args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT },
-        },
-      ],
-    },
-    {
-      domain: 'LENDING_POOL',
-      actions: [
-        {
-          actionType: STORY_ACTIONS.LENDING_POOL.LEND,
-          suiteName: '',
-          args: { pass: true, parent: null },
-        },
-        {
-          actionType: STORY_ACTIONS.LENDING_POOL.WITHDRAW,
-          suiteName: '',
-          args: { pass: true, parent: STORY_ACTIONS.LENDING_POOL.LEND },
-        },
-      ],
-    },
-    {
-      domain: 'DAPP',
-      actions: [
-        {
-          actionType: STORY_ACTIONS.DAPP.LEND,
-          suiteName: '',
-          args: { pass: true, parent: null, dapp: 1 },
-        },
-        {
-          actionType: STORY_ACTIONS.DAPP.SWAP,
-          suiteName: '',
-          args: { pass: true, parent: null, dapp: 0 },
-        },
-      ],
-    },
-  ]
+  // let manualScenarios: TestScenario[] = [
+  //   {
+  //     domain: 'LOAN',
+  //     actions: [
+  //       {
+  //         actionType: STORY_ACTIONS.LOAN.TAKE_OUT,
+  //         suiteName: '',
+  //         args: { pass: true, parent: null },
+  //       },
+  //       {
+  //         actionType: STORY_ACTIONS.LOAN.LIQUIDATE,
+  //         suiteName: '',
+  //         args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT },
+  //       },
+  //       {
+  //         actionType: STORY_ACTIONS.LOAN.REPAY,
+  //         suiteName: '',
+  //         args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT },
+  //       },
+  //     ],
+  //   },
+  //   {
+  //     domain: 'LENDING_POOL',
+  //     actions: [
+  //       {
+  //         actionType: STORY_ACTIONS.LENDING_POOL.LEND,
+  //         suiteName: '',
+  //         args: { pass: true, parent: null },
+  //       },
+  //       {
+  //         actionType: STORY_ACTIONS.LENDING_POOL.WITHDRAW,
+  //         suiteName: '',
+  //         args: { pass: true, parent: STORY_ACTIONS.LENDING_POOL.LEND },
+  //       },
+  //     ],
+  //   },
+  //   {
+  //     domain: 'DAPP',
+  //     actions: [
+  //       {
+  //         actionType: STORY_ACTIONS.DAPP.LEND,
+  //         suiteName: '',
+  //         args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT, dapp: 1 },
+  //       },
+  //       {
+  //         actionType: STORY_ACTIONS.DAPP.SWAP,
+  //         suiteName: '',
+  //         args: { pass: true, parent: STORY_ACTIONS.LOAN.TAKE_OUT, dapp: 0 },
+  //       },
+  //     ],
+  //   },
+  // ]
 
```

Comment:
given that we aren't going to use manual tests in our new testing suite, the above code should be presumed dead code and removed